renderer/renderer.html

```diff
@@ -385,6 +385,10 @@ <h3>Output</h3>
                     <input class="check-input" type="checkbox" value="" id="keepUnmerged">
                     <label class="check-label" for="keepUnmerged">Keep unmerged files</label>
                 </div>
+                <div class="mb-1">
+                    <input class="check-input" type="checkbox" value="" id="avoidFailingToSaveDuplicateFileName">
+                    <label class="check-label" for="avoidFailingToSaveDuplicateFileName">Avoid failing to save duplicate filename</label>
```

Comment:
I dont quite like the name for this, can we call it "Enumate Duplicate Filenames" instead?

Reply:
6e287ec6

Reply:
Now all files will be Enumated if there is a duplicate file name in the folder. so this name wont make sense.

I named it like this as making this active will make the Resumability function inaccessible.

Reply:
I am mostly unhappy with that naming since it makes the default case a "failure", which is not the case.

> Now all files will be Enumated if there is a duplicate file name in the folder. so this name wont make sense.

Surely we only add the (1) if the file already exists, not add a number to every download, no?

Reply:
> I am mostly unhappy with that naming since it makes the default case a "failure", which is not the case.
I mean, it is the case, what's why I am doing this lol. Few of my videos failed to download from a playlist of 300 videos, took me a while to figure them out. 

The logs were not opening too with that playlist, another bug because the log was too big?

>Surely we only add the (1) if the file already exists, not add a number to every download, no?
This is no longer the case since the latest commit. If this option is not active, noting happens. yt-dlp will save the unmerged file in the download folder and merged them in the same folder.

Reply:
> We can also easily add a more description on hover using the title attribute, it will look like this:

Thank you so much. Will document it in the Wiki once this PR is merged.

Reply:
@StefanLobbenmeier

Reply:
Sorry did not check the PR last week, but should be able to get this merged this weekend